library/src/main/java/com/rockerhieu/emojicon/EmojiconGridFragment.java

```diff
@@ -66,8 +67,7 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
             mData = People.DATA;
             mUseSystemDefault = false;
         } else {
-            Object[] o = (Object[]) getArguments().getSerializable("emojicons");
-            mData = Arrays.asList(o).toArray(new Emojicon[o.length]);
+            mData = (Emojicon[]) getArguments().getParcelableArray(EMOJICONS_KEY);
```

Comment:
yes, we see this exception in our exception reporting. (reported on Android 4.1.2 and 4.4.2 so far)